### DISCUSSION TITLE
Element attribute watching or binding?

### DISCUSSION THREAD

**Discussion by deng232** [alpinejs/alpine#3708] [1. Help]
I want a canvas overlay on an image; therefore, I need the canvas size response to the image or image container's size change after the image being loaded.

is there any build in way to do it?

**Comment by ekwoka** [comment]
using CSS, have a 1x1 css grid, and set both children (image and canvas) to be on that 1 cell. They will naturally share size pressures, and be stacked.

**Reply by deng232** [reply]
I have tried css before, it will enlarge the pixel in the canvas rather than increase the amount of pixels

**Reply by ekwoka** [reply]
Well, you could use like a resize observer on it.
